data_validation/cli_tools.py

```diff
@@ -164,10 +164,18 @@ def configure_arg_parser():
         _configure_find_tables(subparsers)
         _configure_raw_query(subparsers)
         _configure_run_parser(subparsers)
+        _configure_deploy(subparsers)
 
     return parser
 
 
+def _configure_deploy(subparsers):
+    """Configure arguments for deploying as a service."""
+    subparsers.add_parser(
+        "deploy", help="(Beta) Deploy Data Validation as a Service (w/ Flask)"
```

Comment:
We added some functionality a while back to support beta CLI commands in PR #249. It would allow a user to run `data-validation beta deploy`

Looks like that functionality is broken. For some reason the original implementation of the beta CLI on [line 124](https://github.com/GoogleCloudPlatform/professional-services-data-validator/pull/249/files#diff-ce217c386e702709267508ebe1bc725106c6bcba326c1ccb29951d506855ae6aR124) of the PR has moved [inside the if loop](https://github.com/GoogleCloudPlatform/professional-services-data-validator/blob/develop/data_validation/cli_tools.py#L150) which is a breaking change.

Could we fix the change and support the deploy argument in the beta CLI?

Something like this:
```
def configure_arg_parser():
    """Extract Args for Run."""
    parser = argparse.ArgumentParser(
        usage=__doc__, formatter_class=argparse.RawDescriptionHelpFormatter
    )

    parser.add_argument("--verbose", "-v", action="store_true", help="Verbose logging")
    parser.add_argument("beta", nargs='?', help="Beta flag to enable beta features for the tool.")

    if "beta" in sys.argv:
        subparsers = parser.add_subparsers(dest="command")
        _configure_deploy(subparsers)
    else:
        subparsers = parser.add_subparsers(dest="command")
        _configure_validate_parser(subparsers)
        _configure_run_config_parser(subparsers)
        _configure_connection_parser(subparsers)
        _configure_find_tables(subparsers)
        _configure_raw_query(subparsers)
        _configure_run_parser(subparsers)

    return parser
```

Reply:
Changing the beta design a little to make it prominent.  Should be resolved now